tests/integration/hpe_3par_manager.py

```diff
@@ -439,7 +452,7 @@ def hpe_verify_volume_created(self, volume_name, vvs_name=None, **kwargs):
         if 'size' in kwargs:
             self.assertEqual(hpe3par_volume['sizeMiB'], int(kwargs['size']) * 1024)
         else:
-            self.assertEqual(hpe3par_volume['sizeMiB'], 102400)
+            self.assertEqual(hpe3par_volume['sizeMiB'], 10240)

```

Comment:
Please confirm if this change is required

Reply:
Reverted back